### PR TITLE
SKILL.tvs 3600-3700

### PR DESCRIPTION
**Pull request by MrLiar**
Basically fix up some lines. Help proof read it please.

003603
003607
003608 determination or confidence 
003610 not sure what it means by high chance to faint
003612 change it to hilt but not sure what to do with the bottom line - no x
003613
003614
003617 fix it upbut it can be better also does the shield fade or no? - no x
003619 would it be critical chance? - no x
003620 base off the videoe it is just a hit with the shield and weapon so i remove continous - no x
003621
003622
003623 reworded it but not sure if it blocks stabs only or other attacks - no x
003624
003625 its says destroy shield but i am assuming it meant defense - no x
003627
003628 reword it but could work on the front - no x
003631 to long?
003632
003633 reword it but not sure what to do with "dark" - no x
003635 reworded - no x
003641
003644 makes no sense but reworded - no x
003647
003648
003654 reworded but does it really stun bosses? - no x
003655
003657 does it fade when party is gone or the tent? - no x
003658
003661
003662
003671
003674
003675
003676
003677
003678 might need to reword "stream" - no x
003679
003680
003681
003682
003683
003684
003685
003686
003687
003688
003689   does it actually freeze narby enemies? - no x
003690  reworded - no x
003691
003692 reworded - no x
003694
003695
003697 reword - no x
003698 rewworded but what status? - no x
003699 what is going on here?

### CODE REVIEW

File: SKILL.tsv [hunk 1]
```diff
@@ -3604,62 +3604,62 @@ SKILL_20150414_003603	Shovelling
...
+xSKILL_20150414_003621	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Hit target with a rising strike knocking them away.
+xSKILL_20150414_003622	{#DD5500}{ol}[Physical] - [Hit]{/}{/}{nl}A heavy blow to the target's head temporarily decreasing INT and SPR.
+SKILL_20150414_003623	Take defensive stance. Enemies blocked becomes weak to stab attacks.
+xSKILL_20150414_003624	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Attack with mutiple strikes.
+SKILL_20150414_003625	{#DD5500}{ol}[Physical] - [Hit]{/}{/}{nl}Make a strong swipe pushing aside the target and destroying its defense.
 SKILL_20150414_003626	{#DD5500}{ol}[Physical] - [Stab]{/}{/}{nl}Repeatedly stab enemy with spear.
-SKILL_20150414_003627	{#DD5500}{ol}[Physical] - [Stab]{/}{/}{nl}Stab enemy with spear. Continuous damage is applied as the size the monster is larger.
-SKILL_20150414_003628	Change to Window formation for active attack. Chance of critical increases but evasion decreases.
+xSKILL_20150414_003627	{#DD5500}{ol}[Physical] - [Stab]{/}{/}{nl}Imapale the target and deal damage overtime according to the size of the monster
```
Comment (vtange) [inline SKILL.tsv]: impale

File: SKILL.tsv [hunk 2]
```diff
@@ -3604,62 +3604,62 @@ SKILL_20150414_003603	Shovelling
...
-SKILL_20150414_003622	{#DD5500}{ol}[Physical] - [Hit]{/}{/}{nl}Smack down enemy's head. Target's INT and SPR temporarily decrease.
-SKILL_20150414_003623	For a defensive stance using weapon. Enemies blocked becomes vulneratble to stab attacks.
-SKILL_20150414_003624	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Swing sword around to make continuous attack.
-SKILL_20150414_003625	{#DD5500}{ol}[Physical] - [Hit]{/}{/}{nl}Strike and push enemy sidewards using the side of sword. Enemy's shield is destroyed.
+SKILL_20150414_003620	{#DD5500}{ol}[Physical] - [Hit] - [Slash]{/}{/}{nl}Use shield and sword to attack.
+xSKILL_20150414_003621	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Hit target with a rising strike knocking them away.
+xSKILL_20150414_003622	{#DD5500}{ol}[Physical] - [Hit]{/}{/}{nl}A heavy blow to the target's head temporarily decreasing INT and SPR.
+SKILL_20150414_003623	Take defensive stance. Enemies blocked becomes weak to stab attacks.
+xSKILL_20150414_003624	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Attack with mutiple strikes.
+SKILL_20150414_003625	{#DD5500}{ol}[Physical] - [Hit]{/}{/}{nl}Make a strong swipe pushing aside the target and destroying its defense.
```
Comment (vtange) [inline SKILL.tsv]: make a strong swipe, pushing

File: SKILL.tsv [hunk 3]
```diff
@@ -3604,62 +3604,62 @@ SKILL_20150414_003603	Shovelling
...
-SKILL_20150414_003620	{#DD5500}{ol}[Physical] - [Hit] - [Slash]{/}{/}{nl}Use shield and sword to make continuous attacks.
-SKILL_20150414_003621	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Use weapon to raise and blow away enemy.
-SKILL_20150414_003622	{#DD5500}{ol}[Physical] - [Hit]{/}{/}{nl}Smack down enemy's head. Target's INT and SPR temporarily decrease.
-SKILL_20150414_003623	For a defensive stance using weapon. Enemies blocked becomes vulneratble to stab attacks.
-SKILL_20150414_003624	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Swing sword around to make continuous attack.
-SKILL_20150414_003625	{#DD5500}{ol}[Physical] - [Hit]{/}{/}{nl}Strike and push enemy sidewards using the side of sword. Enemy's shield is destroyed.
+SKILL_20150414_003620	{#DD5500}{ol}[Physical] - [Hit] - [Slash]{/}{/}{nl}Use shield and sword to attack.
+xSKILL_20150414_003621	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Hit target with a rising strike knocking them away.
+xSKILL_20150414_003622	{#DD5500}{ol}[Physical] - [Hit]{/}{/}{nl}A heavy blow to the target's head temporarily decreasing INT and SPR.
```
Comment (vtange) [inline SKILL.tsv]: head, temporarily

**Comment by MrLiar** [comment]
thanks , updated

**Comment by HilandLogic** [comment]
Faint needs to be changed to Stun

**Comment by imcgames** [comment]
The Pull Request is closed for issues regarding copyrights as stated in our main page. 
We will be updating you with details in the near future regarding this matter so please hold your Pull Requests until things the issue is resolved. 
We apologize for the inconvenience caused.

**Pull request closed without merging**